pkg/resource/coordinator.go

```diff
@@ -79,20 +79,25 @@ func (c *Coordinator) executeTaskFlow(f *task.TaskFlow) {
 			c.Logger.Error(err, "No executable function found for task")
 		} else {
 			taskId := task.GetTaskManager().Submit(taskFunc)
-			c.Logger.Info("Successfullly submit task", "taskid", taskId)
+			c.Logger.Info("Successfully submit task", "taskId", taskId)
 			f.OperationContext.TaskId = taskId
 			f.OperationContext.TaskStatus = taskstatus.Running
 		}
 	case taskstatus.Running:
 		// check task status and update cr status
 		taskResult, err := task.GetTaskManager().GetTaskResult(f.OperationContext.TaskId)
+
 		if err != nil {
 			c.Logger.Error(err, "Get task result got error", "task id", f.OperationContext.TaskId)
+			c.Manager.PrintErrEvent(err)
 			f.OperationContext.TaskStatus = taskstatus.Failed
 		} else {
 			if taskResult != nil {
-				c.Logger.Info("task finished", "task id", f.OperationContext.TaskId, "task result", taskResult.Status)
+				c.Logger.Info("task finished, wait processing task result", "task id", f.OperationContext.TaskId, "task result", taskResult)
 				f.OperationContext.TaskStatus = taskResult.Status
+				if taskResult.Error != nil {
```

Comment:
Output normal-level events may result in excessive Describe commands for API objects, and it may be more appropriate to view the Normal-level (Info level) information in the log.